packages/katello/katello/katello.spec

```diff
@@ -5,7 +5,7 @@
 %global confdir common
 
 Name:       katello
-Version:    3.7.0
+Version:    3.7.1
```

Comment:
We keep this inline with the actual Katello release. So just bump the release and leave this as 3.7.0